docs/proposals/PROPOSAL_A.md

```diff
@@ -0,0 +1,115 @@
+# Proposal A - FreeBSD Jails
+
+This proposal recommends changes to describe the requirements for mapping an OCI
+runtime container to a corresponding FreeBSD jail.
+
+## Modifications
+
+This suggests adding an object to the [FreeBSD-specific section](https://github.com/opencontainers/runtime-spec/blob/main/config.md#platform-specific-configuration) of the [container configuration](https://github.com/opencontainers/runtime-spec/blob/main/config.md) to describe the required parameters for the jail.
+
+## Jail Configuration
+
+Jail parameters to use for the container's jail.
+
+**`jail`** _(object, OPTIONAL)_ jail parameters for this container.
+
+The following parameters can be specified for for the container jail:
+
+- **`parent`** _(string, OPTIONAL)_ - parent jail.
+    The value is the name of a jail which should be this container's parent (defaults to none).
+- **`host`** _(string, OPTIONAL)_ - allow overriding hostname, domainname, hostuuid and hostid.
```

Comment:
From a JSON perspective this is a string but it sounds like the valid values are effectively an enum: "new" and "inherit"?

Reply:
That's correct - in theory there are three vaues, "new", "inherit" and "disable" but I don't use "disable" at the moment at all. The string values match those used by the jail command line tool but they are translated to integers for the jail syscalls. I slightly prefer the strings for this interface for readability.